scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -91,32 +59,36 @@
      :state task-state}))
 
 (defn make-pod-watch-callback
-  [handle-status-update handle-exit-code]
+  [kcc handle-status-update handle-exit-code]
   (fn pod-watch-callback
     [prev-pod pod]
     (try
-      (let [prev-state (pod->pod-state prev-pod)
-            state (pod->pod-state pod)
+      (let [prev-state (api/pod->synthesized-pod-state prev-pod)
+            state (api/pod->synthesized-pod-state pod)
             task-id (-> (or pod prev-pod)
                         .getMetadata .getName)
             namespace (-> (or pod prev-pod)
                           .getMetadata
                           .getNamespace)]
-        (if (= namespace "cook")
+        (if (nil? pod)
+          (controller/pod-deleted kcc prev-pod)
+          (controller/pod-update kcc pod))
+
+        '(if (= namespace "cook")
```

Comment:
What's with the single quote?

Reply:
A way of commenting out that section so that you could salvage it for when you do the handle-status-update refactor/cleanup. I switched it to a standard line comment.